### ISSUE TITLE
MongoDB extension broken on Ubuntu 22.04 (ns_parserr)

### ISSUE THREAD

**Issue by webmozart** [mongodb/mongo-php-driver#1390]
### Bug Report

On a fresh Ubuntu 22.04 on AWS, when I install PHP 7.4 or 8.1 with the MongoDB extension from Ondrej PHP, the extension fails to load with the error `undefined symbol: ns_parserr`.

### Environment

Ubuntu 22.04 with PHP 7.4 or PHP 8.1

```
# php -i | grep -E 'mongodb|libmongoc|libbson'
PHP Warning:  PHP Startup: Unable to load dynamic library 'mongodb.so' (tried: /usr/lib/php/20190902/mongodb.so (/usr/lib/php/20190902/mongodb.so: undefined symbol: ns_parserr), /usr/lib/php/20190902/mongodb.so.so (/usr/lib/php/20190902/mongodb.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
/etc/php/7.4/cli/conf.d/20-mongodb.ini,
```

### Test Script

```bash
# apt-add-repository ppa:ondrej/php
# apt install php7.4 php7.4-mongodb
# php7.4 -v
PHP Warning:  PHP Startup: Unable to load dynamic library 'mongodb.so' (tried: /usr/lib/php/20190902/mongodb.so (/usr/lib/php/20190902/mongodb.so: undefined symbol: ns_parserr), /usr/lib/php/20190902/mongodb.so.so (/usr/lib/php/20190902/mongodb.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0

# apt install php8.1 php8.1-mongodb
# php8.1 -v
PHP Warning:  PHP Startup: Unable to load dynamic library 'mongodb.so' (tried: /usr/lib/php/20210902/mongodb.so (/usr/lib/php/20210902/mongodb.so: undefined symbol: ns_parserr), /usr/lib/php/20210902/mongodb.so.so (/usr/lib/php/20210902/mongodb.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
```

### Expected and Actual Behavior

I expect PHP to load the MongoDB extension without a warning.

### Related Issues

* https://github.com/oerdnj/deb.sury.org/issues/1872
* https://bugzilla.redhat.com/show_bug.cgi?id=2009394

**Comment by jmikola** [comment]
@webmozart: Looking at your OP in https://github.com/oerdnj/deb.sury.org/issues/1872 it appears you're using version 1.14.0 of the extension:

```
Installed: 1.14.0+1.11.1+1.9.2+1.7.5-1+ubuntu22.04.1+deb.sury.org+1
```

It's likely this is related to an earlier bug we fixed in 1.14.2. See: [PHPC-2152](https://jira.mongodb.org/browse/PHPC-2152). That fix also appears in the most recent 1.15.0 release. I'm not sure what @oerdnj's schedule is for updating packages, but the 1.14.2 release is about 1.5 months old at this point.

I'd suggest trying either of those version through PECL and seeing if the issue persists.

**Comment by webmozart** [comment]
Fixed here: https://github.com/oerdnj/deb.sury.org/issues/1872
